front/src/modules/scenario/components/ScenarioCard.tsx

```diff
@@ -14,16 +14,16 @@ import { updateSelectedProjection } from 'reducers/osrdsimulation/actions';
 import { useAppDispatch } from 'store';
 import { dateTimeFormatting } from 'utils/date';
 
-type StudyCardProps = {
+type ScenarioCardProps = {
   setFilterChips: (filterChips: string) => void;
   scenario: ScenarioWithCountTrains;
 };
 
-export default function StudyCard({ setFilterChips, scenario }: StudyCardProps) {
+export default function ScenarioCard({ setFilterChips, scenario }: ScenarioCardProps) {
   const { t } = useTranslation('operationalStudies/study');
   const navigate = useNavigate();
   const dispatch = useAppDispatch();
-  const { updateScenarioID } = useOsrdConfActions();
+  const { updateScenarioID } = useOsrdConfActions(); // Update après
```

Comment:
You forgot to remove this comment
```suggestion
  const { updateScenarioID } = useOsrdConfActions();
```

Reply:
My bad, I was tired